content/people/damiri.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Delaram Amiri
+role: student-phd
+status: Ph.D. Candidate
+year joined: 2015
+portrait: damiri.jpg
+badge: damiri-badge.jpg
+interests:
+    - Healthcare IoT
+    - Sensor Control
+    - Optimization
+    - Edge Computing
+education:
+    - degree: M.Sc. in Electrical Engineering, Communications
+        institution: Indiana University - Purdue University Indianapolis
+        year: 2015
+    - degree: B.Sc. in Electrical Engineering, Communications
+        institution: Shiraz University
+        year: 2013
+email: damiri@uci.edu
+websites:
+    - title: LinkedIn
+    url: www.linkedin.com/in/delaramamiri
+bibliography:
+    - title: Journal Articles items: - delaram1 -delaram2 -delaram3 -delaram4
```

Comment:
Fix the `items` array formatting in each of the bibliography sections.
(See the example file for reference.)